### ISSUE TITLE
AsyncAPI rule to validate property example in schema

### ISSUE THREAD

**Issue by aleung** [stoplightio/spectral#1732]
**User story.**
As a API designer, I can run spectral to check my AsyncAPI, so that I can find out invalid property example in schema.

**Describe the solution you'd like**
Current Spectral AsyncAPI ruleset has a `asyncapi-schema-examples` rule to validate `examples` under a schema. However it doesn't check the `example` of a property. I'd expect there is an AsyncAPI rule that is equivalent to OAS rule `oas3-valid-example`.

**Additional context**
Below is an AsyncAPI snippet with invalid example:

``` yaml
components:
  schemas:
    id:
      type: integer
      example: "123456"
```

**Comment by magicmatatjahu** [comment]
@aleung Hi! I don't think this issue is still valid, because AsyncAPI 2.x.x never supported the `example` field in Schema Object (documentation https://github.com/asyncapi/spec/blob/master/spec/asyncapi.md#properties). AsyncAPI Schema Object has only `examples` field, which is currently being validated - https://github.com/stoplightio/spectral/blob/develop/docs/reference/asyncapi-rules.md#asyncapi-schema-examples

**Comment by aleung** [comment]
@magicmatatjahu Thank for the clarification. I checked the specification and indeed the schema object singular example was an OpenAPI-specific keyword.
